src/js/app.js

```diff
@@ -1 +1,3 @@
 // JS Goes here - ES6 supported
+
+import { nav } from './nav-collapsible';
```

Comment:
Because the nav code can be cut down so much, and the js class added to the body should happen outside of a nav specific module, and you're not actually importing anything (see my comments on your nav code for reference), I'd switch this to something like: